tests/DCPS/UnitTests/Serializer.cpp

```diff
@@ -341,6 +343,186 @@ TEST(serializer_test, Serializer__swap_bytes_endianness)
   EXPECT_EQ(ser.endianness(), ENDIAN_NONNATIVE);
 }
 
+TEST(serializer_test, Serializer__align_context_basic_reference)
```

Comment:
```suggestion
TEST(serializer_test, Serializer_align_context_basic_reference)
```
(and others)
Identifiers containing a double-underscore are reserved.

Reply:
Should the other tests above also not have double-underscore?

Reply:
Nothing in the entire repo "should" have it.  But for this PR we can limit the scope to files or functions you're already changing.